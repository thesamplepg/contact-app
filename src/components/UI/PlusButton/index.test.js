import React from "react";
import { shallow } from "enzyme";

import Component from "./index";

describe("ContactList component", () => {
  const renderComponent = () => shallow(<Component />);

  it("render propertly", () => {
    expect(renderComponent()).toMatchSnapshot();
  });
});
