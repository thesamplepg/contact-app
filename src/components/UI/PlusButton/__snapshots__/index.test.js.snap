// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactList component render propertly 1`] = `
<button
  className="PlusButton"
>
  <div />
  <div />
</button>
`;
